website.js

```diff
@@ -319,6 +316,17 @@ io.on('connection', function (socket) {
             socket.emit('nextSong');
         }})
     })
+    socket.on('createPoll', function (data) {
+        connection.query('insert into poll set ?', data, function(err, result) {})
+    })
+    socket.on('getPoll', function (data) {
+        connection.query('select * from poll where id = ?', data, function(err, result) {
+            socket.emit('pollData', result[0].answers)
+        })
+    })
+    socket.on('addResult', function (data) {
+        connection.query('update poll set answers = ? where id = "' + data.id + '"', JSON.stringify(data.answers), function(err,result){})
```

Comment:
I've been looking into this all night and got it working up until showing the results. Appreciate the help, Will commit the things I've implemented in the next few days with working results.